src/script_lua.c

```diff
@@ -1513,17 +1526,17 @@ void luaRegisterRedisAPI(lua_State* lua) {
      * This is not a deep copy but is enough for our purpose here. */
     lua_getglobal(lua,SERVER_API_NAME);
     lua_setglobal(lua,REDIS_API_NAME);
-    /* Replace math.random and math.randomseed with our implementations. */
+    
+    /* Replace math.random and math.randomseed with custom implementations. */
     lua_getglobal(lua,"math");
-
+    /* Add random and randomseed functions. */
```

Comment:
```suggestion
```
I feel like this comment isn't saying anything.

Reply:
yeah there were many such comments, so instead of removing I just made them a full sentence.

Reply:
removed.